src/renderer/ipcRegistration.ts

```diff
@@ -58,3 +58,16 @@ ipc.on('initiate-undo', async () => {
 ipc.on('initiate-redo', async () => {
   dispatchRedo();
 });
+
+/**
+ * Used by backend to initiate export from front end
+ */
+ipc.on('initiate-export-project', async () => {
+  // Retrieve current project state from redux
+  const { currentProject } = store.getState();
+
+  // Don't export if we don't have a project open
+  if (currentProject === null) return;
+
```

Comment:
Have a look at 'initiate-save-project'. I think export needs to be split into 2 separate functions. 1 - to get the save file path from the backend (then dispatch that to redux). 2 - to export the project using the backend.

Reply:
I think the logic is identical here for both atm. Unless I missed something?

Reply:
![image](https://user-images.githubusercontent.com/101784287/168513884-6e38b16b-7bed-4e57-831e-f0e16a90a37b.png)

Reply:
wait so exportEDL should be called here? and not in [exportProjectToFile](https://github.com/patrickbrett/mlvet/pull/56/files#:~:text=export%20const%20exportProjectToFile%3A%20()